ci/scripts/PKGBUILD

```
@@ -69,6 +69,12 @@ build() {
   mkdir -p ${cpp_build_dir}
   pushd ${cpp_build_dir}
 
+  # We use static cURL in google-cloud-cpp. If we can use cURL's CMake
+  # package, we don't need to specify CURL_STATICLIB explicitly. But
+  # we don't have cURL's CMake package. We need to use CXXFLAGS
+  # instead of ARROW_CXXFLAGS because ARROW_CXXFLAGS aren't passed to
+  # ExternProjects.
```

Add conditional so it's easier to understand (at least for me :) )
```suggestion
  # We use static cURL in google-cloud-cpp. If we could use cURL's CMake
  # package, we wouldn't need to specify CURL_STATICLIB explicitly. But
  # we don't have cURL's CMake package. We need to use CXXFLAGS
  # instead of ARROW_CXXFLAGS because ARROW_CXXFLAGS aren't passed to
  # ExternProjects.
```

---

Thanks! English suggestions are always helpful to me! I'll use your suggestion!